code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -160,6 +167,10 @@
 					break
 
 		if("preview")
+			if(shuttle_and_preview_cooldown + PREVIEW_OR_SHUTTLE_SPAWN_COOLDOWN > world.time) 
+				to_chat(usr, "<span class='warning'>Please wait while the shuttle manipulator recalibrates...</span>")
```

Comment:
This is an OOC thing used by admins, dont have to make it IC-centric

Reply:
I'll remove the diegetic stuff and get more to the point